### ISSUE TITLE
port CI changes to release branches

### ISSUE THREAD

**Issue by beckermr** [LSSTDESC/CCL#913]
The changes in the CI scripts in https://github.com/LSSTDESC/CCL/pull/912 likely need to be ported to release branches.

**Comment by c-d-leonard** [comment]
Sorry this is probably silly but I don't really understand what you mean by release branches. The releases are made off ther master branch - what am I missing? @beckermr

**Comment by beckermr** [comment]
There are branches that are named release/xyz. We should port the changes that fixed read the docs and the ci scripts we made late last year to these branches. 

Sorry for writing such a vague issue!

**Comment by carlosggarcia** [comment]
I don't think we are using these kind of branches anymore. Unless you think we should, I'll close this issue.
